internal/nostr/models.go

```diff
@@ -14,26 +14,30 @@ const (
 	NIP_47_RESPONSE_KIND   = 23195
 
 	// state of request event
-	REQUEST_EVENT_PUBLISH_CONFIRMED   = "confirmed"
-	REQUEST_EVENT_PUBLISH_FAILED      = "failed"
+	REQUEST_EVENT_PUBLISH_CONFIRMED = "CONFIRMED"
+	REQUEST_EVENT_PUBLISH_FAILED    = "FAILED"
```

Comment:
are these still used and is it ok to change the casing? will the previous values be manually migrated?

Reply:
Yup, we can do it this way instead of writing a migration

```SQL
UPDATE request_events
SET state = UPPER(state);
```

Reply:
@im-adithya keep in mind for an open source project we typically also have to think about other people who might run this. 
it's fine here, but keep this in mind.